spec/syntax.md

```diff
@@ -733,4 +733,18 @@ s = 1*( SP / HTAB / CR / LF )
 
 The grammar is formally defined in [`message.abnf`](./message.abnf)
 using the ABNF notation,
-as specified by [RFC 5234](https://datatracker.ietf.org/doc/html/rfc5234).
```

Comment:
Nit: since 7405 is "structured as a set of changes to the full ABNF
   specification [RFC5234]", should we technically keep both references here? Or is 7405 enough?